src/vignette/util/image_response.clj

```diff
@@ -15,6 +15,11 @@
          add-surrogate-header
          surrogate-key)
 
+(defmacro when-header-val
+  ([resp key val]
+   `(if ~val
+     (header ~resp ~key ~val) ~resp)))
+
```

Comment:
[Don't write a macro if a function will do it.](https://github.com/bbatsov/clojure-style-guide#macros).